.github/workflows/check_smoke_tests.yml

```diff
@@ -71,7 +71,7 @@ jobs:
         -H "Content-Type: application/json")
 
         echo "Response: $RESPONSE_2"
-        echo "$RESPONSE_2" | jq -e '.is_match == true'  
+        echo "$RESPONSE_2" | jq -e '.prediction == match'  
```

Comment:
```suggestion
        echo "$RESPONSE_2" | jq -e '.prediction == "match"'  
```

Reply:
I think after changing each of these to a string the smoke tests should be good. Since `.person_reference_id` still exists on the response object that part should be the same. If any other errors come up with the smoke tests after these changes I'll take another look!